app/views/schemes/new.html.erb

```diff
@@ -65,15 +65,13 @@
                                       "data-controller": %w[accessible-autocomplete conditional-filter] %>
       <% end %>
 
-      <% support_services_provider_selection = Scheme.support_services_providers.map do |key, value| %>
-        <% OpenStruct.new(id: value, name: key.to_s.humanize) %>
-      <% end %>
+      <% mantype_selection = Scheme.arrangement_types.keys.excluding("Missing").map { |key, _| OpenStruct.new(id: key, name: key.to_s.humanize) } %>
 
-      <%= f.govuk_collection_radio_buttons :support_services_provider_before_type_cast,
-                                           support_services_provider_selection,
+      <%= f.govuk_collection_radio_buttons :arrangement_type,
+                                           mantype_selection,
                                            :id,
                                            lambda { |option|
-                                             if option.id.zero? && !current_user.support?
+                                             if option.id == "The same organisation that owns the housing stock" && !current_user.support?
                                                "Your organisation"
                                              else
                                                option.name
```

Comment:
Changing this is more involved than first thought - potentially worth reworking in a future PR?

Reply:
What about leaving the keys and value strings here but making the checks something like:

```ruby
if Scheme::ARRANGEMENTTYPE[option.id].zero?
```

Reply:
The problem I was having with keys and values was that the option selected wouldn't remain selected if you hit save and continue to move on and then hit the back button. I'm not sure if arrangement type can be compared to zero since the enum maps to strings 🤔

Reply:
Right well `== "0"`  or `to_i` then. The example was just the idea that we look up the thing we're comparing to from a single source of truth rather than duplicating the comparison string across the code base.

Reply:
I think I've got your suggestion implemented @baarkerlounger 🙂